monitoring/helm/monitoring/values.yaml

```diff
@@ -140,8 +140,6 @@ kube-prometheus-stack:
     enabled: false
 
   defaultRules:
-    additionalRuleLabels:
```

Comment:
I think this might be needed to visualize these alerts in the console currently, I'm not entirely sure though

Reply:
We need to improve the way the console gets alerts anyways so that it can properly filter them for namespace, pod name, container name, etc.

Reply:
It already autolinks to an app if the namespace is there, so if removing this just repins them to the original app it might just work

Reply:
The namespace label gets added by the alert so this just ensure it doesn't get overridden causing the namespace label to be `monitoring` for every alert.